tests/src/Kernel/ValueObject/ImageValueObjectTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\ValueObject;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
+use Drupal\Tests\token\Kernel\KernelTestBase;
+
+/**
+ * Test image value object with image field type. Extracted from oe_theme.
+ *
+ * @see https://github.com/openeuropa/oe_theme/blob/3.x/tests/src/Kernel/ValueObject/ImageTest.php
+ */
+class ImageValueObjectTest extends KernelTestBase {
+  private const ALT_TITLE = 'This is an alternative title';
+  private const TITLE = 'This is a Title';
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'image',
+    'file',
+    'entity_test',
+    'field',
+  ];
+
+  /**
+   * Entity object.
+   *
+   * @var \Drupal\Core\Entity\EntityInterface
+   */
+  private $entity;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installEntitySchema('file');
+    $this->installEntitySchema('entity_test');
+
```

Comment:
Nit: drop this empty line. Keep the entity schema installations together. I know we are copying it over, but we can fix this.